include/test/CDataFrameAnalysisSpecificationFactory.h

```diff
@@ -15,6 +15,7 @@
 #include <api/CDataFrameAnalysisSpecification.h>
 #include <api/CDataFrameTrainBoostedTreeRegressionRunner.h>
 
+#include <rapidjson/document.h>
 #include <test/ImportExport.h>
```

Comment:
nit: these should be flipped around and separated because we include our own headers first and group by library.

```suggestion
#include <test/ImportExport.h>

#include <rapidjson/document.h>
```